alkos/kernel/arch/x86_64/linker.ld

```diff
@@ -22,17 +22,17 @@ SECTIONS
 	/* First put the multiboot header, as it is required to be put very early
 	   in the image or the bootloader won't recognize the file format.
 	   Next we'll put the .text section. */
-          .multiboot ALIGN(64) : AT(.)
-          {
-	          KEEP(*(.multiboot))
+    .multiboot ALIGN(64) : AT(.)
+    {
+        KEEP(*(.multiboot))
 	}
 
 	. = ALIGN(4K);
 
 	.text32 BLOCK(4K) : ALIGN(4K)
-          {
-                    *(.text32)
-          }
+    {
```

Comment:
i believe misaligning this is a mistake, but ehh not really that important

Reply:
I think it was wrongly aligned before so I fixed it